Sources/SwiftDocCUtilities/Docc.swift

```diff
@@ -10,10 +10,12 @@
 
 import ArgumentParser
 
+// TODO: Append `Docc.Init.self` depending on value of `isExperimentalDocCInitCommandEnabled` feature flag
```

Comment:
I think this is the right approach but it's going to be difficult to do with a command line flag since the list of subcommands are created before the command line arguments have been parsed. To support this the init command would probably need to be gated by a `defaults` value on macOS and an environmental variable on other  platforms. 

I don't think we have any current examples of this but we had some on the 5.9 branch, for example in [LinkResolutionMigration.swift](https://github.com/apple/swift-docc/blob/release/5.9/Sources/SwiftDocC/Infrastructure/Link%20Resolution/LinkResolutionMigration.swift)

Reply:
After discussing this with Daniel I decided to remove the feature flag since all the logic is self-contained and doesn't affect the others DocC commands.

Resolved in https://github.com/apple/swift-docc/pull/719/commits/2a993c4e59d8222a3359af61a8b4ef0b3be9bdcf